certified-connectors/Zoho Mail/apiDefinition.swagger.json

```diff
@@ -1274,8 +1274,7 @@
                         "title": "Folder name"
                       },
                       "folderId": {
-                        "type": "integer",
```

Comment:
The "type" has been changed from integer to string, can you please confirm that no existing flows or users will be affected by this change?

Reply:
yes, it won't affect the existing user.